components/Header/src/stories/bem.stories.mdx

```diff
@@ -9,528 +9,522 @@ import "../index.scss";
+                fill="#4B4B4B"
+              ></path>
+              <path
+                fill-rule="evenodd"
+                clip-rule="evenodd"
+                d="M36.275 24.6831C36.1176 24.5578 35.9516 24.4252 35.7804 24.2881V25.7264L35.7072 25.7354L33.3436 24.6166L32.5703 24.7992L35.5178 26.1982L35.8202 26.5878L36.305 26.5182L36.488 26.0591L36.275 25.7852V24.6831Z"
+                fill="#4B4B4B"
+              ></path>
+              <path
+                fill-rule="evenodd"
+                clip-rule="evenodd"
+                d="M34.1618 12.5193C34.1618 12.5193 33.5983 12.7713 33.4603 12.7713C33.323 12.7713 33.1943 12.6583 33.1943 12.5193C33.1943 12.38 33.323 12.2671 33.4603 12.2671C33.5983 12.2671 34.1618 12.5193 34.1618 12.5193Z"
+                fill="black"
+              ></path>
+              <path
+                fill-rule="evenodd"
+                clip-rule="evenodd"
+                d="M28.3912 15.643C28.0378 15.0175 28.4515 14.6351 28.7421 14.3415C28.9522 14.128 29.1343 13.9449 29.1382 13.6754C29.1435 13.3217 28.9424 13.2571 28.4479 13.1854C28.1541 13.1426 27.8503 13.0985 27.6773 12.8757C27.5796 12.7484 27.5397 12.5999 27.5627 12.4467C27.614 12.1047 27.9648 11.8491 28.0043 11.8211C28.1035 11.7517 28.2387 11.7766 28.3069 11.8758C28.3757 11.9751 28.3516 12.1117 28.2531 12.1812C28.1754 12.2365 28.0103 12.3895 27.992 12.5127C27.9887 12.5373 27.9895 12.5657 28.0205 12.6059C28.0847 12.6896 28.3103 12.7221 28.5093 12.7512C28.9373 12.8133 29.5838 12.9072 29.5724 13.6817C29.5661 14.1289 29.2918 14.4066 29.0498 14.6516C28.7742 14.93 28.5495 15.1336 28.6985 15.5267C28.7962 15.7845 29.2383 15.9211 29.1534 16.0737C29.0733 16.2174 28.5644 15.9496 28.3912 15.643Z"
+                fill="black"
+              ></path>
+              <path
+                fill-rule="evenodd"
+                clip-rule="evenodd"
+                d="M37.4742 25.6321C37.4742 25.6321 37.2824 25.4818 36.9793 25.2422V26.1489H36.9851L36.7399 26.5771L36.9793 26.9967V30.5471L37.4742 30.3222V27.0053L37.7194 26.5771L37.4742 26.1489V25.6321Z"
+                fill="black"
+              ></path>
+              <path
+                fill-rule="evenodd"
+                clip-rule="evenodd"
+                d="M41.1624 26.4463L40.9131 26.7787L41.1624 26.4463Z"
+                fill="#636363"
+              ></path>
+              <path
+                fill-rule="evenodd"
+                clip-rule="evenodd"
+                d="M41.0719 19.83C40.3176 19.0498 39.4234 18.4218 38.4293 18.0034L38.1799 18.3358C39.1622 19.005 39.8087 20.0049 40.197 20.6256C40.9172 21.7474 41.6887 23.3928 42.1551 25.5793C42.257 26.019 42.3404 26.4833 42.3992 26.9764L43.4386 27.7048C43.9687 24.7001 42.9619 21.7848 41.0719 19.83Z"
+                fill="#636363"
```

Comment:
@robbieatacato moeten all deze fills niet `currentColor` hebben?

Reply:
@bleddynacato ik kon geen png toevoegen, dus vandaar voor storybook een svg.
Dit is een logo, en hoeft niet met css de kleur bepaald te worden.
In de WordPress thema gebruiken we een png, zie de staging site.